andela/travel_tool_front
# #162045684 Enable travel team members verify requests

#### What does this PR do?
Provides user interface for travel team members to view approved requests and verify them

#### Description of Task to be completed?
- travel team members can view list of verified and approved requests for their centers
- travel team members can see a modal with a button to click to verify a request
- requester gets an in-app notification stating the request has been verified

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ft-verify-request-162045684`
- Start the server with `make start`
- Clone the backend repo `git clone https://github.com/andela/travel_tool_back.git`
- checkout the branch `git checkout ft-verify-request-162045684`
- Start the server with `make start`
- Visit https://travela-local.andela.com
- Login to the application
- Ensure you have a travel team member and manager privilege
- Create a request and approve it.
- Fill the required checklist items.
- Navigate to the My Verifications page under the Request menu on the left side nav.
- You should see a page with a list of requests pending verification.
- Click on a request to review and verify it

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162045684](https://www.pivotaltracker.com/story/show/162045684)

#### Screenshots (if appropriate)
![verify-requests](https://user-images.githubusercontent.com/13545380/48983409-d159b500-f0ee-11e8-819f-d7e0d36a9290.gif)

<img width="662" alt="screen shot 2018-11-26 at 1 30 12 pm" src="https://user-images.githubusercontent.com/13545380/49014161-79b75a00-f17f-11e8-86de-6014913fb6e9.png">


#### Questions:
